Added check for ordinal position not matching source and destination

royc1 opened the pull request:
Added check for Key value ordinal position not matching source and destination for Gptransfer

While running gptransfer with --partition-table option, user got KeyError: 83 as key value was not present in the dictionary. Therefore, added a check if the key value is not present in the dictionary then it will return error.

danielgustafsson commented:
Can you please provide a description of what this change fixes?

Chibin commented:
Looks good

hlinnaka commented:
@royc1, can you answer @danielgustafsson's question, please, so that we can dislodge this PR from the list of open PRs?

royc1 commented:
While running gptransfer with --partition-table option, user got KeyError: 83 as key value was not present in the dictionary. Therefore, added a check if the key value is not present in the dictionary then it will return error.  This comment I already added in the description.

royc1 commented:
Pushed

danielgustafsson commented:
Thanks for the clarification. In reference for future pull requests, please rewrite the commit message with the extended information as well since this PR will be lost and forgotten while the commit is the entity which will live on in the repository. Sorry for not being clear on my wish in the previous comment.
